Makefile

```diff
@@ -42,7 +42,7 @@ ifeq ($(OPERATING_SYSTEM),Darwin)
 	brew install cppcheck
 	brew install include-what-you-use
 	brew install llvm
-	ln -s "$(brew --prefix llvm)/bin/clang-tidy" "/usr/local/bin/clang-tidy"
+	test -f /usr/local/bin/clang-tidy || sudo ln -s "$(shell brew --prefix llvm)/bin/clang-tidy" "/usr/local/bin/clang-tidy"
```

Comment:
the why on these changes:

- don't need to add a symlink every time this is ran.
- need to be in a privileged account to add symlinks to /usr/local/bin
- can't get the data from a command without using shell